remill/Arch/AArch64/Arch.cpp

```diff
@@ -719,6 +719,28 @@ static bool DecodeBitMasks(uint64_t N /* one bit */,
   }
   return true;
 }
+/// Utility function for extracting [From, To] bits from a uint32_t.
+static inline unsigned slice(uint32_t Bits, unsigned From, unsigned To) {
```

Comment:
Make this take in `uint64_t`s and return a `uint64_t`s to avoid downstream casts.

Reply:
Lower case variable names. `Bits` -> `bits`.

Reply:
Uppercase function names, `slice` -> `Slice`.